knx/dpt/formats.go

```diff
@@ -92,20 +108,15 @@ func unpackU8(data []byte, i *uint8) error {
 
 func packU32(i uint32) []byte {
 	buffer := []byte{0, 0, 0, 0, 0}
-	buffer[1] = uint8(i >> 24)
-	buffer[2] = uint8(i >> 16)
-	buffer[3] = uint8(i >> 8)
-	buffer[4] = uint8(i & 0xff)
-	return buffer
+	binary.BigEndian.PutUint32(buffer[1:], i)
+	return buffer[:]
```

Comment:
What's the benefit of copying the buffer before returning?

Reply:
In that case, none. It was a copy / paste from the code of another application. I revert it.